<ui version="4.0" >
 <class>PermissionsPluginWidget</class>
 <widget class="QWidget" name="PermissionsPluginWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QCheckBox" name="checkPermissions" >
     <property name="text" >
      <string>Change &amp;Permissions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupPermissions" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Access Permissions</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="1" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>O&amp;wner:</string>
          </property>
          <property name="buddy" >
           <cstring>comboPermOwner</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="comboPermOwner" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text" >
            <string>Forbidden</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Can Read</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Can Read &amp; Write</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Gro&amp;up:</string>
          </property>
          <property name="buddy" >
           <cstring>comboPermGroup</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QComboBox" name="comboPermGroup" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text" >
            <string>Forbidden</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Can Read</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Can Read &amp; Write</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Ot&amp;hers:</string>
          </property>
          <property name="buddy" >
           <cstring>comboPermOthers</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QComboBox" name="comboPermOthers" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text" >
            <string>Forbidden</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Can Read</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Can Read &amp; Write</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0" colspan="2" >
         <widget class="QLabel" name="labelAdvanced" >
          <property name="text" >
           <string>This file uses advanced permissions</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkFolder" >
        <property name="text" >
         <string>Is Executable/Only owner &amp;can rename and delete folder content</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton" >
          <property name="text" >
           <string>A&amp;dvanced Permissions</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkOwner" >
     <property name="text" >
      <string>&amp;Change Owner</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupOwner" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Ownership</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;User:</string>
        </property>
        <property name="buddy" >
         <cstring>comboUser</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="comboUser" />
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>&amp;Group:</string>
        </property>
        <property name="buddy" >
         <cstring>comboGroup</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="comboGroup" />
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
